src/screens/Settings/components/Tools/index.tsx

```diff
@@ -18,6 +20,14 @@ export default function Tools({ appName, runner }: Props) {
   const { t } = useTranslation()
   const [winecfgRunning, setWinecfgRunning] = useState(false)
   const [winetricksRunning, setWinetricksRunning] = useState(false)
+  const [progress, setProgress] = useState<string[]>([])
+  const winetricksOutputBottomRef = useRef<HTMLDivElement>(null)
+  const logRef = useRef<HTMLDivElement>(null)
+  const [autoScroll, setAutoScroll] = useState(true)
+
+  ipcRenderer.on('progressOfWinetricks', (e, messages) => {
+    setProgress(messages)
+  })
```

Comment:
I think we have to do this in a `useEffect(() => ...., [])` so we can remove the listener when this is unmounted